test/mixtures.jl

```diff
@@ -22,6 +23,10 @@ using LinearAlgebra
     @test TS.numtype(tN) == TaylorN{Float64}
     @test normalize_taylor(tN) == tN
     @test tN.order == 3
+    @test HomogeneousPolynomial([1]) > xH > yH > 0.0
```

Comment:
I think a (nested) `@testset` would be useful to indicate that these lines intend to show compliance with the reference by Berz would be useful.

Reply:
I agree. But the whole tests are quite messy, so let me postpone this for another PR...

Reply:
I know the rest is messy, but let's lot make a Herculean task of ever cleaning them byt not making a small start here!